content/tutorials/kit-assembly.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Connecting your kit
+---
+
+## The bits you should have
+- Raspberry Pi (the board in a plastic case with many USB ports)
+- Power Board (the board in a plastic case with many green sockets and two buttons)
+- Motor Board (the other board in a plastic case)
+- Arduino (The blue, exposed board with a metal USB-B connector)
+- Arduino Servo Shield (another blue board, hopefully on top of the first)
+- 1 x 3.81mm Camcon (for the Arduino Servo Shield)
```

Comment:
I think saying something like "green connector" instead of "Camcon" would be much clearer (despite being less precise and a bit of a mouthful). Not sure though - thoughts?

Reply:
Alternatively, maybe it's better to just define what a "camcon" is near the top of the document (a picture would be nice too).

Reply:
Resolved by 65e281d503b7582c24d00ad43aaf27220a331fca.